static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -247,10 +247,23 @@ const STATES = {
     PAUSED: onPlayerPaused,
 };
 
+const isUSLabsSponsored = (): boolean => {
+    const isUSContent =
+        config.get('page.productionOffice', '').toLowerCase() === 'us';
+    const isSponsored = config.get('page.sponsorshipType') === 'paid-content';
+    const isGlabsUS =
+        config.get('page.trackingNames', '').toLowerCase() === 'glabs us';
```

Comment:
This might not be something we can rely on...
We should be able to use `config.page.isPaidContent` to identify Labs content.
If the value is `false` when we expect it to be `true` then there might be a bigger issue for commercial dev to look at?

Reply:
I have spoken with Alex and it seems like the flag `isPaidContent` was not been set correctly for this particular video and hopefully it will be resolved soon and will return true. It doesn't look a generic issue as for other glabs articles it does return true. Going forward, i have now change it to rely on `isPaidContent` flag